### DISCUSSION TITLE
Highlights are not visible in code files and gists

### DISCUSSION THREAD

**Discussion by ecc521** [community/community#53299] [Repositories]
### Select Topic Area

Bug

### Body

Since ~2 months ago, I have been unable to see the text that I highlight (to copy) in GitHub code files and gists. This makes it extremely difficult to accurately copy code portions, and I usually have to view the raw file to see what I'm copying. 

Utilizing latest Chrome version on MacOS Ventura.

**Comment by rmdhfz** [comment]
Hi @ecc521 ,

You can try a few things to resolve the issue:

1. Clear your browser cache and cookies. Sometimes, outdated data can cause display issues.
2. Disable any browser extensions you have installed, especially those related to syntax highlighting or code display.
3. Try viewing the code files in a different browser, such as Firefox or Safari.
4. Check if there are any updates available for your browser, and install them if possible.

If none of these solutions work, you can also try reaching out to GitHub support for further assistance.

**Comment by ishan1m** [comment]
Happens to me too. Not sure what is causing it.

**Comment by ishan1m** [comment]
@ecc521 Oh this fixed it for me https://github.com/orgs/community/discussions/50295#discussioncomment-5353843
